src/Arcus.Observability.Tests.Unit/Telemetry/ILoggerExtensionsTests.cs

```diff
@@ -178,22 +183,26 @@ public void LogHttpDependency_ValidArguments_Succeeds()
             var logger = new TestLogger();
             HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, _bogusGenerator.Internet.Url());
             HttpStatusCode statusCode = _bogusGenerator.PickRandom<HttpStatusCode>();
-            DateTimeOffset startTime = _bogusGenerator.Date.PastOffset();
-            var duration = _bogusGenerator.Date.Timespan();
-
-            // Act;
-            logger.LogHttpDependency(request, statusCode, startTime, duration);
 
-            // Assert
-            var logMessage = logger.WrittenMessage;
-            Assert.StartsWith(MessagePrefixes.DependencyViaHttp, logMessage);
-            Assert.Contains(request.RequestUri.Host, logMessage);
-            Assert.Contains(request.RequestUri.PathAndQuery, logMessage);
-            Assert.Contains(request.Method.ToString(), logMessage);
-            Assert.Contains(((int)statusCode).ToString(), logMessage);
-            Assert.Contains(startTime.ToString(CultureInfo.InvariantCulture), logMessage);
-            var isSuccessful = (int)statusCode >= 200 && (int)statusCode < 300;
-            Assert.Contains($"Successful: {isSuccessful}", logMessage);
+            using (var measurement = DependencyMeasurement.Start())
```

Comment:
We need both approaches to be tested actually

Reply:
Ok